bundles/com.espressif.idf.debug.gdbjtag.openocd/META-INF/MANIFEST.MF

```diff
@@ -26,7 +26,9 @@ Require-Bundle: ilg.gnumcueclipse.debug.gdbjtag,
  org.eclipse.cdt.build.gcc.core,
  com.espressif.idf.launch.serial.core;bundle-version="1.0.0",
  com.espressif.idf.ui;bundle-version="1.0.0",
- ilg.gnumcueclipse.core
+ ilg.gnumcueclipse.core,
+ org.eclipse.cdt.debug.ui;bundle-version="8.4.200",
```

Comment:
Could you remove the version number for this? Eclipse CDT 2020-09 comes with 8.4.100 and is supported by IEP

Reply:
Done